lib/wanda_web/auth/access_token.ex

```diff
@@ -0,0 +1,16 @@
+defmodule WandaWeb.Auth.AccessToken do
+  @moduledoc """
+    Jwt Token is the module responsible for creating a proper jwt access token.
+    Uses Joken as jwt base library
+  """
+
+  use Joken.Config, default_signer: :access_token_signer
+
+  @iss "https://github.com/trento-project/web"
```

Comment:
Shouldn't have these 2 in the config file as in the web project?
It should use `wanda` url by the way.

Reply:
The issuer is the web since it acts as an identity provider.

I don't think those two are configurations, even if the web they are. In fact, I wanted to change  web, because we don't need them to be dynamic.

Reply:
Ok, I see that it shares the same issuer with the web as this 1st one is the one responsible of issuing keys for wand too.